apps/studio/src/features/sign-in/components/LoginStep/InitialLoginStep.tsx

```diff
@@ -1,18 +1,34 @@
-import { Stack, Text } from "@chakra-ui/react"
+import { Box, Stack, Text, VStack } from "@chakra-ui/react"
 
+import { IsomerLogo } from "~/components/Svg"
 import { useEnv } from "~/hooks/useEnv"
+import { useIsSingpassEnabled } from "~/hooks/useIsSingpassEnabled"
 import { EmailLoginForm } from "../EmailLogin"
 
 export const InitialLoginStep = (): JSX.Element => {
   const {
     env: { NEXT_PUBLIC_APP_NAME: title },
   } = useEnv()
+  const isSingpassEnabled = useIsSingpassEnabled()
 
   return (
-    <Stack gap="2rem" direction="column" width="100%">
-      <Text color="base.content.brand" textStyle="h3-semibold">
-        {title}
-      </Text>
+    <Stack gap="1.5rem" direction="column" width="100%">
+      <Box>
+        <IsomerLogo />
+      </Box>
```

Comment:
abit confused - why do we need this extra `Box` here (but no impact la tbh)

Reply:
Hmm yeah for some reason not having this `Box` will cause the logo to become very short and cut off.